Source/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ add_definitions(-D__STDC_CONSTANT_MACROS)
 
 macro(add_dolphin_library lib srcs libs)
 	add_library(${lib} STATIC ${srcs})
-	target_link_libraries(${lib} ${libs})
+	target_link_libraries(${lib} PUBLIC ${libs})
```

Comment:
Well, the default behaviour is PUBLIC. Changing it to PRIVATE would change semantics and probably break a lot of things currently. I can try it quickly and report how it breaks!